Updated the mail domain for regression test against new BVTMail server.

Blackbaud-MohitAhir opened the pull request:
Updated the mail domain for regression test against new BVTMail server.

Blackbaud-MohitAhir committed:
Updated the mail domain for regression test against new BVTMail server.

Blackbaud-GagandeepGujral commented on lib/watirmark_email/qamail.rb:
```diff
@@ -1,6 +1,6 @@
 module WatirmarkEmail
   class QAMail < BaseController
-    URL           = "qasendmail.conviocloud.com"
+    URL           = "bvtmail.conviocloud.com"
```
shouldn't this be bvtmail.blackbaudcloud.com? Same for the other one?

Blackbaud-MohitAhir replied:
Yes. Updated it.

Blackbaud-MohitAhir committed:
Updated the email server domain

Updated the email server domain from qasendmail.conviocloud.com to bvtmail.blackbaudcloud.com.
